src/ripple/app/misc/Validations.cpp

```diff
@@ -456,22 +465,27 @@ class ValidationsImp : public Validations
         if (mWriting)
             return;
 
-        mWriting = true;
-        app_.getJobQueue ().addJob (
-            jtWRITE, "Validations::doWrite",
-            [this] (Job&) { doWrite(); });
+        // addCountedJob() may return false (Job not added) at shutdown.
+        mWriting = app_.getJobQueue ().addCountedJob (
+            jtWRITE, "Validations::queuedWrite",
+            [this] (Job&) { queuedWrite(); });
     }
 
-    void doWrite ()
+    void queuedWrite ()
     {
         auto event = app_.getJobQueue ().makeLoadEvent (jtDISK, "ValidationWrite");
```

Comment:
`event` is an odd duck.  We're instantiating a `LoadEvent` here, which is a scoped timing kind of thing.  It adds information to the LoadMonitor when it is created and destroyed.  So it's doing work, it just doesn't look like it 😃.